tests/bulk_create/models.py

```diff
@@ -6,6 +6,11 @@ class Country(models.Model):
     iso_two_letter = models.CharField(max_length=2)
 
 
+class Union(models.Model):
```

Comment:
Ideally we would the `m2m` field to one of the existing models of this test application to prevent the creation of an extra table on test database initialization. What about adding a self referential `ManyToMany` to `Country` named `neighboring`?